vip_hci/hci_dataset.py

```diff
@@ -454,7 +549,7 @@ def stats(self, region='circle', radius=5, xy=None, annulus_inner_radius=0,
         _ = frame_quick_report(self.image, self.fwhm, source_xy, verbose)
 
 
-class HCIDataset:
+class HCIDataset(object):
```

Comment:
Should we inherit from `object` in all classes for Python 2 compatibility?

Reply:
Yes. In python2, by default all classes are old-style classes. To make them behave like in python3, we have to explicitly specify classes as new-style by inheriting from `object`. E.g. `super()` is only for new-style classes.